### PR TITLE
Update technical spec link and title on home page

### PR DESCRIPTION
**Pull request by danmitchell-**
Requested by Jen Lambourne.

### CODE REVIEW

File: src/main/java/uk/gov/register/configuration/HomepageContent.java [hunk 1]
```diff
@@ -26,7 +26,7 @@ public String getUsingRegistersGuidanceUrl() {
 
     @SuppressWarnings("unused, used from template")
     public String getSpecificationUrl() {
-        return "http://openregister.github.io/specification/";
+        return "http://open-registers-docs.readthedocs.io/en/latest/";
```
Comment (chr1sdi) [inline src/main/java/uk/gov/register/configuration/HomepageContent.java]: any reason why not HTTPS?

**Comment by danmitchell-** [comment]
I don't know what you mean @chr1sdi ;-)

**Comment by chr1sdi** [comment]
master of time ;)
👍

**Pull request merged**